sbi/utils/torchutils.py

```diff
@@ -222,16 +226,22 @@ def ensure_parameter_batched(parameter: torch.Tensor) -> torch.Tensor:
     return parameter
 
 
-# XXX does an in-place version (e.g. make_conform_) make sense?
+# XXX does an in-place version (e.g. ensure_observation_batched_) make sense?
 def ensure_observation_batched(observation: torch.Tensor) -> torch.Tensor:
     """
-    Return tensors that both have the same tensor.ndim
-    Function also covers cases where parameters is ndim=1 and observation ndim=2
-    Also, we have specialized check for multi-dimensional data x, e.g. images.
+    Return observation tensor that has a batch dimension, i.e. has shape
+    (1, dim_observation)
+
+    Function also covers cases where observation has observation.ndim>1, e.g. images
+
+    Args:
+        observation: A single observation, potentially without batch dimension
+    Returns:
+        Batched observation
     """
 
-    # => ensure observation has shape (1, dim_observation). If shape[0] > 1, we assume that the batch-dimension
-    # is missing, even though ndim might be >1 (e.g. for images)
+    # => ensure observation has shape (1, dim_observation). If shape[0] > 1, we assume
+    # that the batch-dimension is missing, even though ndim might be >1 (e.g. images)
```

Comment:
There's no need for a dash here. Although this is not really important in a code comment, it is important when writing papers. Dashes are needed to make a compound term (two-word phrase) act as a modifier (in the role of an adjective).